<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FillSinkDialogBase</class>
 <widget class="QDialog" name="FillSinkDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FillSink</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOutput">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>35</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevation Grid :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Output Grid :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLayers">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenDialog">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>70</y>
     <width>80</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Shape :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbShape">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
